client/src/legacy/TinyMCE_sslink.js

```diff
@@ -109,7 +109,7 @@ jQuery.entwine('ss', ($) => {
 
       const attributes = this.buildAttributes(data);
       const sanitise = createHTMLSanitiser();
-      const linkText = sanitise(data.Text);
+      const linkText = data.Text === undefined ? '' : sanitise(data.Text);
```

Comment:
I think it would make more sense to apply the fix to the `createHTMLSanitiser()` method in `ShortcodeSerialiser.js` - that way it'll fix any other cases of this problem at the same time.

Reply:
Cool, I've changed it.